com.github.joseexposito.touche.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.joseexposito.touche
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: com.github.joseexposito.touche
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=home'
+  - '--filesystem=host:ro'
```

Comment:
What do you need this for?

Reply:
Hi, and thanks for your quick review. You guys are doing an amazing work!

This app is a GUI to configure [Touchégg](https://github.com/JoseExposito/touchegg)
I need to access `/usr/share/touchegg/touchegg.conf` in order to load the default configuration and, in case that the file is not present, show a window indicating that Touchégg is not installed and a link to the downloads page.

Reply:
Right, and the home one? :)

Reply:
Sorry, I thought you mean host :sweat_smile: 

Once the default configuration is loaded (the one in `/usr/share`) the user can modify it using the UI.
That new configuration is then saved in `~/.config/touchegg/touchegg.conf`.

Reply:
You can make that specific instead of having the whole home access :)

Reply:
Thanks for the advice, updated